tool/gravity/cli/run.go

```diff
@@ -70,6 +70,12 @@ func Run(g *Application) (err error) {
 		return trace.Wrap(err)
 	}
 
+	if *g.Debug {
+		utils.InitGRPCLoggerWithDefaults()
+	} else {
+		utils.InitGRPCLoggerFromEnvironment()
```

Comment:
So the debug flag and GRPC logger env variables cannot be used together?

Reply:
True that - I've restructured the code so it was not the case.